tests/v2_validation/cattlevalidationtest/core/test_network_policy.py

```diff
@@ -0,0 +1,287 @@
+from common_fixtures import *  # NOQA
+
+test_network_policy = os.environ.get(
+    'TEST_NETWORK_POLICY', "False")
+np_reason = \
+    'Rancher NetworkPolicy Manager needs to de deployed in "Default" env'
+if_network_policy = pytest.mark.skipif(test_network_policy != "True",
```

Comment:
@leodotcloud , I have changed the existing reason string to - ```Intended to not execute network policy test``` . Added check_for_network_policy_manager() method to create_env_for_network_policy so that it prevents the setup and  any test from getting executed when network_policy_manager stack  is not deployed in the setup.